lib/util/esWrapper.js

```diff
@@ -153,7 +159,7 @@ class ESWrapper {
         break;
       }
     }
-
+    console.log(error)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)